cmd/minishift/cmd/completion.go

```diff
@@ -32,15 +32,16 @@ const longDescription = `
 
 	This depends on the bash-completion binary.  Example installation instructions:
 	OS X:
-		$ brew install bash-completion
-		$ source $(brew --prefix)/etc/bash_completion
-		$ minishift completion bash > ~/.minishift-completion  # for bash users
-		$ minishift completion zsh > ~/.minishift-completion  # for zsh users
-		$ source ~/.minishift-completion
-	Fedora:
-		$ dnf install bash-completion
-		$ minishift completion bash > ~/.minishift-completion  # for bash users
-		$ source ~/.minishift-completion
+			$ brew install bash-completion
```

Comment:
This seems to be lot of extra spaces.

I think for better readability we can write as: 

```
This depends on the bash-completion binary.  Example installation instructions:

OS X:

    $ brew install bash-completion
    $ source $(brew --prefix)/etc/bash_completion
    $ minishift completion bash > ~/.minishift-completion  # for bash users
    $ minishift completion zsh > ~/.minishift-completion  # for zsh users
    $ source ~/.minishift-completion

RHEL/CentOS/Fedora:

    $ yum install bash-completion # for RHEL/CentOS
    $ dnf install bash-completion   # for Fedora
    $ minishift completion bash > ~/.minishift-completion  # for bash users
    $ source ~/.minishift-completion
```

Reply:
Single tab would be file for steps indentation